stan/math/opencl/kernel_generator/load.hpp

```diff
@@ -49,7 +49,7 @@ class load_
    * Creates a deep copy of this expression.
    * @return copy of \c *this
    */
-  inline load_<T&> deep_copy() const & { return load_<T&>(a_); }
+  inline load_<T&> deep_copy() const& { return load_<T&>(a_); }
```

Comment:
I think it was the autoformatter, but I dunno why it looked at this file.

Reply:
Here is the commit: https://github.com/stan-dev/math/commit/d539b587942685abca1fa4901de432e5dbbb9da7#diff-e92a842ba1a127d7587a635913cb4ce3 . Since this is the autoformatter I'm not going to bother with trying to revert it. Presumably it'd just put it back again.

Reply:
Yes, this is related to https://github.com/stan-dev/math/issues/1822

The styling on develop was forced by an older version of clang-format.